### PR TITLE
CTCP-3682: Update notification schema to include enrollmentEORINumber

### PR DESCRIPTION
**Pull request by drnaylor-oc**

### CODE REVIEW

File: source/documentation/developer-setup.html.md.erb [hunk 1]
```diff
@@ -226,6 +226,10 @@ Message received push notifications must conform to the following JSON schema.
       "type": "string",
       "description": "The type of message being received."
     },
+    "enrollmentEORINumber": {
+      "type": "string",
+      "description": "The enrollment EORI number that owns the movement. Replaces customerID from version 1."
```
Comment (drnaylor-oc) [inline source/documentation/developer-setup.html.md.erb:231]: Not sure if this is the best way to word this, or if mentioning customerID here is even the right thing to do, but I figured I'd add it -- thoughts are welcome.

Comment (fmcgrath) [reply source/documentation/developer-setup.html.md.erb:231]: The first sentence is OK.

However, the second sentence needs to be tweaked.

Suggestion:
(This parameter replaces `customerId` in CTC Traders API v1.0.)

Question: I can't find any mention of "customer" or "customerId" in API definition for v1 of API - is that a glitch?

Observation: On second thoughts, does the description in the snippet need to mention customerId or v1 of API? The API changelog text covers that issue and that should be sufficient.

File: source/documentation/developer-setup.html.md.erb [hunk 2]
```diff
@@ -345,6 +351,10 @@ Submission notification push notifications must conform to the following JSON sc
       "type": "string",
       "description": "ID of the message that is being submitted."
     },
+    "enrollmentEORINumber": {
+      "type": "string",
+      "description": "The enrollment EORI number that owns the movement. Replaces customerID from version 1."
+    },
```
Comment (fmcgrath) [inline source/documentation/developer-setup.html.md.erb:357]: See comment above.

**Review by fmcgrath — COMMENTED** [review]
See first comment. Two sentences might be overkill.

File: source/documentation/developer-setup.html.md.erb (hunk 1 above)
Comment (drnaylor-oc) [reply source/documentation/developer-setup.html.md.erb:231]: > I can't find any mention of "customer" or "customerId" in API definition for v1 of API - is that a glitch?

No as it was only on the push notifications, and this really shouldn't have been named "customerId" in v1 anyway.

> On second thoughts, does the description in the snippet need to mention customerId or v1 of API? The API changelog text covers that issue and that should be sufficient.

The changelog is historical and shouldn't really be used for migrating from version 1 to 2 -- this sort of information really belongs in some sort of migration guide. So while I agree that this sentence shouldn't necessarily be here, if we have it, it needs to be somewhere along with any other migration information and not (solely) the changelog.

Comment (fmcgrath) [reply source/documentation/developer-setup.html.md.erb:231]: OK. Let's go with my suggestion for now. The issue of a migration guide might UR and some discussion. If we were to add such content to, say, the NCTS5 Traders API service guide, we would need to cover at least several migration issues and not just this one.

**Review by fmcgrath — APPROVED** [review]
Looks OK.

**Pull request merged**